data/mods/Aftershock/recipes/recipes.json

```diff
@@ -319,21 +319,6 @@
     "//": "Conversion of a normal smartphone into an atomic one by replacing the power supply.",
     "components": [ [ [ "plastic_chunk", 1 ] ], [ [ "processor", 1 ] ], [ [ "smart_phone", 1 ] ], [ [ "plut_cell", 1 ] ] ]
   },
-  {
-    "type": "recipe",
-    "activity_level": "fake",
-    "result": "purifier_smart_shot",
```

Comment:
You need to obsolete this recipe.
It's just an entry like this:
```json
{
  "type": "recipe",
  "result": "purifier_smart_shot",
  "obsolete": true
}
```

Reply:
Ohhh. What's the difference, from the game's perspective?

Reply:
I am guessing it's to handle characters who already know it.

Reply:
Yep, it gives an error on load if you know it without this.

Reply:
Ahh, good to know, thanks!